### DISCUSSION TITLE
Next available VLAN ID

### DISCUSSION THREAD

**Discussion by sdrenner** [netbox-community/netbox#14078] [Ideas]
Is there a way to see the next available VLAN ID, without having to scroll down and look at each number for the next available?
If not I think i'll make a feature request.
Example: so when you go to prefixes and click on one of your main subnets and then click on "child prefixes" and scroll down you can see right away subnets that are available.
TIA.
Steve

**Comment by DamienUKCEH** [comment]
yes it would be useful to supply an end point with the site ID and then get then next vlan number for the site, we could use that in ansible then in the same way we use get next ip for a subnet.
